src/components/Modal/index.android.tsx

```diff
@@ -15,19 +15,17 @@ AppState.addEventListener('blur', () => {
 
 // Only want to use useNativeDriver on Android. It has strange flashes issue on IOS
 // https://github.com/react-native-modal/react-native-modal#the-modal-flashes-in-a-weird-way-when-animating
-function Modal(props) {
+function Modal(props: BaseModalProps) {
     return (
         <BaseModal
-            useNativeDriver
             // eslint-disable-next-line react/jsx-props-no-spreading
             {...props}
+            useNativeDriver
```

Comment:
Any reason for overwriting this prop? What if props has useNativeDriver=false?

Reply:
I must've added it by mistake, I removed it 😄